deliver/lib/deliver/app_screenshot.rb

```diff
@@ -293,13 +302,27 @@ def self.devices
       }
     end
 
+    def self.resolve_ipadpro_conflict_if_needed(device_type, filename)
+      is_3rd_gen = filename.include?("(3rd generation)")
```

Comment:
I though about asking you to actually check for `iPad Pro (12.9-inch) (3rd generation)` instead of just `(3rd generation)`, but as the `device_type` is also checked it doesn't really matter, does it?

Reply:
Yes, filename is checked only if screen size matches iPad Pro's